### DISCUSSION TITLE
Google Summer of Code Proposal Format and Rules

### DISCUSSION THREAD

**Discussion by nthnieljson** [cncf/mentoring#627] [Q&A]
Hello, is there any proposal template and rules specific to CNCF that I need to follow? For example the proposal length limits.

**Comment by 0904-mansi** [comment]
> Hello, is there any proposal template and rules that I need to follow? For example the proposal length limits.
Hey @nthnieljson you can follow the official guide for writing a proposal here https://google.github.io/gsocguides/student/writing-a-proposal 
hope it helps!

**Reply by nthnieljson** [reply]
Thank you for the answer. Just want to confirm, so is there a specific CNCF format or rules? Because that link refers to the general proposal guide for GSoC

**Reply by 0904-mansi** [reply]
> Thank you for the answer. Just want to confirm, so is there a specific CNCF format or rules? Because that link refers to the general proposal guide for GSoC

No there is no particular format available for CNCF. I asked from mentors and they suggested us to follow the official guide.

**Reply by nthnieljson** [reply]
Okay, thanks for clarifying!
